packages/varlet-cli/src/shared/ejsRender.ts

```diff
@@ -0,0 +1,58 @@
+import ejs from 'ejs'
+import fs from 'fs-extra'
+import path from 'node:path'
+import prettier from 'prettier'
+
+export const camelize = (s: string): string => s.replace(/-(\w)/g, (_: any, p: string) => p.toUpperCase())
+
+export const bigCamelize = (s: string): string => camelize(s).replace(s.charAt(0), s.charAt(0).toUpperCase())
+export async function ejsRender(filePath: string, options: any): Promise<void> {
+  try {
+    // 根目录template 绝对路径
+    const templatePath = path.resolve(__dirname, `../../template/create`)
+    // 获取当前渲染文件的 各种 参数 such as ext
+    const file = path.parse(filePath)
+    // 编译根目录 创建的根目录
+    const dest = path.resolve(`${process.cwd()}/src`, options.projectName)
+    // 当前 需要编译的 ejs文件
+    const readFilePath = path.resolve(dest, file.dir, `${file.name}.ejs`)
```

Comment:
You should unify coding style.
In Line 16 to 18. the `path.resolve` write with two style. It's weird.

Reply:
I'm sorry. I don't know how to change this one.